doc/api/tty.md

```diff
@@ -158,7 +158,109 @@ The `tty.isatty()` method returns `true` if the given `fd` is associated with
 a TTY and `false` if it is not, including whenever `fd` is not a non-negative
 integer.
 
+## TTY keybindings
```

Comment:
Can we document these in the readline doc? While a TTY is required for most of these, they shouldonly affect readline behaviour

Reply:
I guess it would be fine to add a reference to those in the `repl` and in here.

Reply:
@addaleax  @BridgeAR just to be clear, I'll add the table of keybindings in the `readline` doc and ref the table in `REPL` and `TTY` doc.

Reply:
@shobhitchittora Yes, that’s the idea (although I don’t really understand why this would be added to the `TTY` doc)

Reply:
@addaleax I saw this code in readline - 

```js
Interface.prototype.write = function(d, key) {
  if (this.paused) this.resume();
  this.terminal ? this._ttyWrite(d, key) : this._normalWrite(d);
};
```
and thought that TTY case need a doc. But yeah we can have the table in `readline` doc and ref it in `REPL` and / or `TTY`.

Need you opinion on this.

Reply:
Yeah, exactly – it’s code in readline. :) We should point out that these codes apply only for TTYs, but it’s not something that’s interesting for code that uses TTYs in general.

Reply:
@addaleax you are right, the TTY docs are actually the wrong place for this. The `repl` should have a reference however.